File: problem 16/Option 01/BoxTest.vb
Public Class Box
    Private _length As Double
    Private _width As Double
    Private _height As Double

    Public Sub New(ByVal l As Double, ByVal w As Double, ByVal h As Double)
        _length = l
        _width = w
        _height = h
    End Sub

    Public Function CalculateVolume() As Double
        Return _length * _width * _height
    End Function
End Class


Module BoxText
    Sub Main()
        Dim box1 As New Box(10, 20, 30)
        Dim volume1 As Double = box1.CalculateVolume()
        Console.WriteLine("Volume of box1: " & volume1)
    End Sub
End Module

File: problem 16/Option 01/BoxTest.swift
class Box {
    var length: Double
    var width: Double
    var height: Double

    init(length: Double, width: Double, height: Double) {
        self.length = length
        self.width = width
        self.height = height
    }

    func calculateVolume() -> Double {
        return length * width * height
    }
}


class BoxText {
    static func main() {
        let box1 = Box(length: 10.0, width: 20.0, height: 30.0)
        let volume1 = box1.calculateVolume()
        print("Volume of box1: \(volume1)")
    }
}


